flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/ResourceManager.java

```diff
@@ -340,7 +344,8 @@ public RegistrationResponse apply(RegistrationResponse registrationResponse, Thr
 		final UUID resourceManagerLeaderId,
 		final String taskExecutorAddress,
 		final ResourceID resourceID,
-		final SlotReport slotReport) {
+		final SlotReport slotReport)
+	{
```

Comment:
Formatting changes are discouraged. Especially if they change the style such that it diverges from the project's implicit norm.